_layouts/default.html

```diff
@@ -4,10 +4,18 @@
 
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, user-scalable=no">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 
 	<title>{% if page.title %}{{ page.title }} · {% endif %}{{ site.title }}</title>
 	<meta name="description" content="{{ site.description }}">
+	<meta name="application-name" content="The Lounge">
+	<meta name="theme-color" content="#415364">
+	<meta property="twitter:card" content="summary">
```

Comment:
Is that one enough? Don't we need the other `twitter:*` too?

Reply:
It falls back to og tags.